src/components/MapTiff/MapTiff.tsx

```diff
@@ -232,35 +223,64 @@ const MapTiff = ({ isReduced = false, ...props }: { isReduced?: boolean }) => {
     [map, tiffs],
   );
 
-  const cleanOcaLayers = (map: maplibregl.Map) => {
-    const mapLayers: string[] = map
-      ?.getLayersOrder()
-      .filter((layer: string) => layer.startsWith("@oca/"));
-    mapLayers.forEach((layer: string) => {
-      if (map?.getLayer(layer)) {
-        map?.removeLayer(layer);
-      }
-    });
-  };
+  const initializeMap = useCallback(() => {
+    if (mapContainer.current) {
+      const newMap = new maplibregl.Map({
+        container: mapContainer.current!,
+        style: MAP_TIFF_STYLE,
+        center: isReduced ? [-54.69, -15.13] : [-51.55, -15],
+        zoom: isReduced ? 3 : 3.6,
+        minZoom: isReduced ? 3 : 3.6,
+        maxZoom: 10,
+      });
 
-  useEffect(() => {
-    if (map && id && year) {
-      if (!isReduced) addPopupEffect(id, year);
-      loadMapLayer(id, year);
+      newMap.on("load", async () => {
+        if (!isReduced)
```

Comment:
With the new change you actually removed the property of not adding the popup effect on the  MapTiff's reduced version, I think just adding it again could solve the issue